bot/cogs/commands.py

```diff
@@ -547,16 +615,30 @@ async def set_main_guild(self, ctx, guild_id: int = None):
     @xp.command(name="give")
     async def xp_give(self, ctx, target: commands.UserConverter, amount: float):
         profile = levelling.UserProfile(target.id, ctx.guild, self.bot)
-        await profile.give_xp(amount)
-        await self.bot.reply_to_msg(ctx.message, f"Gave {amount} xp to {target.name}. "
-                                                 f"They are now rank {profile.rank} ({profile.xp_count} xp)")
+        if not await profile.give_xp(amount):
+            await self.bot.reply_to_msg(ctx.message,
+                                        f"<:thonk:836648850377801769> attempt to give a negative\n"
```

Comment:
Just :thonk: should work, have you tried it ?

Reply:
It won't, I did try. This works 100% of the time across servers.